### PR TITLE
Move test input files from test/e2e/node-selection/ to test/e2e/testing-manifests/ 

### LINKED ISSUE #25398: PodAffinity SchedulerPredicates tests are broken.
because we turned pod affinity feature off. @davidopp 

@david-mcmahon - this is the thing that's killing serial suite.

### PR DESCRIPTION
**Pull request by davidopp**
since the latter are included in the test tarball while the former are not.

Fixes #25398

cc/ @kevin-wangzefeng

### CODE REVIEW

**Comment by kevin-wangzefeng** [comment]
Looks good to me, thanks!

File: test/e2e/scheduler_predicates.go [hunk 1]
```diff
@@ -718,7 +718,7 @@ var _ = framework.KubeDescribe("SchedulerPredicates [Serial]", func() {
 
 		By("Trying to launch a pod that with NodeAffinity setting as embedded JSON string in the annotation value.")
 		labelPodName := "with-labels"
-		nodeSelectionRoot := filepath.Join(framework.TestContext.RepoRoot, "test/e2e/node-selection")
+		nodeSelectionRoot := filepath.Join(framework.TestContext.RepoRoot, "test/e2e/testing-manifests")
```
Comment (ixdy) [inline test/e2e/scheduler_predicates.go]: this should be `test/e2e/testing-manifests/node-selection`, right?

Comment (davidopp) [inline test/e2e/scheduler_predicates.go]: Oops, you're right -- fixed now.

**Comment by ixdy** [comment]
LGTM

**Comment by jlowdermilk** [comment]
@k8s-bot unit test this issue: #25594

**Comment by jayunit100** [comment]
I think we need to update gobindata for this I'll look into it

**Pull request merged**